torchvision/transforms/_functional_video.py

```diff
@@ -1,114 +0,0 @@
-import warnings
-
-import torch
-
-
-warnings.warn(
-    "The 'torchvision.transforms._functional_video' module is deprecated since 0.12 and will be removed in 0.14. "
```

Comment:
I commented this on the internal [diff](https://www.internalfb.com/diff/D39726995) that @YosuaMichael opened to check the deprecation but for visibility to the community here is my comment here:
> we can not deprecate torchvision.transforms._transforms_video since we don't offer currently an alternative. I will be working on this next week. I recommend changing the deprecation message on OSS to notify that we would be removing this on a future version and then go from scratch through a deprecation process once we have a clear alternative to define.

Reply:
@datumbox I add back this file and change the warnings message, could you double check if this message is good? We can change the message to give exact version when it will be removed once we have a clear alternative as you said.

Reply:
I was wondering if it's worth making this change on a separate PR on main because this one is quite massive. The message looks OK to me. There are a few more things we would need to revert such as the unit-tests for the specific module.

Reply:
I'm not sure we really need to delay the removal here: on top of being deprecated, these APIs are private. Users who are still relying on it have been ignoring strong signals. IMHO we should either:

- contact all downstream users who are still relying on this and tell them they have 10 days (or whatever) to update their code
- be nice to them and submit diffs so that they vendor `_functional_video.py` themselves.

Reply:
For the unit-test, do you mean the `test_transforms_video.py`? I also reverted it in this PR so it is not deleted.
For changing the message, I think it is a good idea to have separate PR just to make sure it has more visibility for the change in deprecation plan. I will revert back the message here and create another PR for that then.

Reply:
@NicolasHug in my understanding, currently there is no alternatives for the `_functional_video.py`. In this situation, it will be very difficult for the users to update their code by just changing API or import.

By `they vendor _functional_video.py themselves` do you mean to copy and paste the _function_video.py on users code?

Reply:
@NicolasHug Multiple internal users are still depending on this. Unless you want to dedicate your next 10 days porting their code away from these definitions and working with each individual team to ensure the results in production are identical, I don't think we can proceed with the removal. 

If you check Yosua's [Diff](https://www.internalfb.com/diff/D39726995) you will see multiple tests failing. In addition to the internal stuff, libraries such as [PyTorch Video](https://github.com/facebookresearch/pytorchvideo/blob/430768adbb8e279ca4192db125583a231f3d60c5/pytorchvideo/neural_engine/hook.py) (cc @haooooooqi @lyttonhao) and [ClassyVision](https://github.com/facebookresearch/ClassyVision/blob/35c8f33f2edf4d08d7660bfb61ce4b938aa7aa02/classy_vision/dataset/transforms/util_video.py) (cc @jdsgomes @nairbv) are also affected.

IMO the issue with the specific deprecation is that it doesn't provide an actionable alternative that works out of the box for users and that lead to them not moving away from it. Though I agree there are concerns about respecting our own policies, we probably want to avoid breaking all downstream projects. I recommend reverting this removal to unblock the rest of the deprecations that are more straightforward and bring this topic up for discussion on the next weekly. There we can decide if we should move forwards with deprecating the specific part or not, without delaying the rest of the work.

Reply:
> By they vendor _functional_video.py themselves do you mean to copy and paste the _function_video.py on users code?

Yes, I mean that each downstream library could simply copy this file in their code-base. [` _functional_video.py`](https://github.com/pytorch/vision/blob/main/torchvision/transforms/_functional_video.py) is a small self-contained file that only has `torch` as dependency, so this would be a very reasonable ask from us.

> I recommend reverting this removal to unblock the rest of the deprecations that are more straightforward and bring this topic up for discussion on the next weekly. There we can decide if we should move forwards with deprecating the specific part or not, without delaying the rest of the work

I'm not sure I understand what we mean by "move forwards with deprecating the specific part or not", but I agree we can discuss how to move forward with ` _functional_video.py` case separately (urgently, considering the approaching release).

Reply:
yes I confirm that _ClassyVision uses `_transforms_video.py`

Reply:
If I understand correctly, we are agree to discuss how to handle `_functional_video.py` (probably in weekly). And if we decide to remove it, I will create separate PR for that.